engine/src/hopeit/dataobjects/__init__.py

```diff
@@ -86,6 +86,10 @@ class BinaryAttachment:
     """Type descriptor for multipart binary attachments"""
 
 
+class BinaryDownload:
```

Comment:
We'll need to ensure content-type is always present and tip the users to overwrite it. 
One way could be:

```
@dataclass
class BinaryDownload:
   content-type: str = 'application/json'

```

Reply:
Also document this class explaning with an example how to extend it.